File: logistica/app/model/Equipos.js
Ext.define('Muller.model.Equipos', {
    extend: 'Ext.data.Model',
    idProperty: 'EquipoId',
    autoLoad: false,

    fields: [
    { name:'EquipoId', type:'int' },
    { name:'EquipoNum', type:'string' },
    { name:'EquipoPlaca', type:'string' },
    { name:'EquipoSerial', type:'string' },
    { name:'EquipoCreadoPor', type:'string', defaultValue: Ext.JSON.decode(Ext.util.Cookies.get("CurrentUser")).UsuarioId },
    { name:'EquipoFechaCreado', type:'date', defaultValue: new Date() },
    { name:'EquipoModificadoPor', type:'string', useNull: true, defaultValue: null },
    { name:'EquipoFechaModificado', type:'date', useNull: true },
    { name:'x_EquipoPlaca', type:'string'},
    { name:'x_Estatus', type:'string'}
    ],

    proxy:{
        type:'rest',
        url:'../wa/api/Equipos',
        headers: {
           'Authorization-Token': Ext.util.Cookies.get('MullerAuth')
        },
        reader:{
            type:'json',
            root:'data',
            totalProperty: 'total',
            successProperty: 'success',
            messageProperty: 'message',
            idProperty: 'EquipoId'
        },
        afterRequest: function (request, success) {
            if (request.action == 'read') {
                //this.readCallback(request);
            }
            else if (request.action == 'create') {
                if (!request.operation.success)
                {
                    Ext.popupMsg.msg("Alerta", "Error al grabar el registro");
                    Ext.global.console.warn(request.proxy.reader.jsonData.message);
                } else {
                    Ext.popupMsg.msg("Informaci\u00F3n","Cambios grabados correctamente");
                }
            }
            else if (request.action == 'update') {
                if (!request.operation.success)
                {
                    Ext.popupMsg.msg("Alerta", "Error al grabar el registro");
                    Ext.global.console.warn(request.proxy.reader.jsonData.message);
                } else {
                    Ext.popupMsg.msg("Informaci\u00F3n","Actualizado Correctamente");
                }
            }
            else if (request.action == 'destroy') {
                if (!request.operation.success)
                {
                    Ext.popupMsg.msg("Alerta", "El registro no se elimin\u00F3 correctamente");
                    //Ext.global.console.warn(request.proxy.reader.jsonData.message);
                } else {
                    Ext.popupMsg.msg("Informaci\u00F3n","Eliminado Correctamente");
                }
            }
        }
    },


});

File: MullerWA/MullerWA/Models/Equipos/Equipos.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Data;
using System.Data.SqlClient;

namespace MullerWA.Models
{
    public class Equipo
    {
        public int EquipoId { get; set; }
        public string EquipoNum { get; set; }
        public string EquipoPlaca { get; set; }
        public string EquipoSerial { get; set; }
        public string EquipoCreadoPor { get; set; }
        public DateTime EquipoFechaCreado { get; set; }
        public string EquipoModificadoPor { get; set; }
        public Nullable<DateTime> EquipoFechaModificado { get; set; }
        public string x_EquipoPlaca { get; set; }
        public string x_Estatus { get; set; }
    }

    public class EquipoVisor
    {
        public string Estatus { get; set; }
        public decimal Cantidad { get; set; }
        public decimal PLANTA { get; set; }
        public decimal FERRARI { get; set; }
    }
}